### DISCUSSION TITLE
Question: Is inline assembly allowed?

### DISCUSSION THREAD

**Discussion by rzuckerm** [PlummersSoftwareLLC/Primes#923] [Q&A] [answered]
There is a [discussion](https://openeuphoria.org/forum/137856.wc?last_id=137880) in the OpenEuphoria forum where the author of the Phix language is proposing a solution that has inline assembly. Is this allowed? In my opinion, even though this is a language feature, it seems like cheating.

**Comment by rbergen** [comment, marked as answer]
I converted this to a Q&A discussion because it isn't an issue.

A solution incorporating another language could possibly be submissible as a mixed language solution, not as a solution for the enveloping language (Phix, in this case). For your reference, other mixed language solutions do exist [under the PrimeMixed directory](https://github.com/PlummersSoftwareLLC/Primes/tree/drag-race/PrimeMixed).
